net-misc/omnisync/omnisync-1.0_p20200130-r1.ebuild

```
@@ -14,7 +14,6 @@ SRC_URI="https://gitlab.com/grknight/omnisync/-/archive/${SNAPSHOT}/omnisync-${S
 
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE=""
 
 RDEPEND="net-libs/gnutls:= net-analyzer/net-snmp:="
```

```suggestion
RDEPEND="
	net-analyzer/net-snmp:=
	net-libs/gnutls:=
"
```
i think that would look nicer, also dependencies should be sorted.